packages/sanity/src/core/field/diff/components/ChangeList.tsx

```diff
@@ -133,7 +143,7 @@ export function ChangeList({diff, fields, schemaType}: ChangeListProps): React.R
               <Button
                 tone="critical"
                 mode="ghost"
-                text="Revert all changes"
+                text={t('core.review-changes.revert-button-revert-all')}
```

Comment:
This button and the button on line 130 didn't used to have the same text. However, as I was doing this, it feels like in the spirit of
1. keeping the number of translated keys down (given how minimal the change is) 
2. the spirit of the buttons are basically the same

I made the button have the same text, however, this has visual implications which are not great (check the table at the bottom). I'm tempted to simply revert the change and make them two separate keys as they were before, however, I'm sure this question will come up somewhat frequently as I go forward 🤔  Thoughts? (cc @rexxars @mikolajdobrucki )

| Before | After |
|--------|-------|
|  <img width="684" alt="image" src="https://github.com/sanity-io/sanity/assets/6951139/c23b4451-43ff-48ba-a90f-204517497cc2"> |     <img width="684" alt="image" src="https://github.com/sanity-io/sanity/assets/6951139/16bb2a2d-b371-4ae3-a0db-c6536122a061"> |

Reply:
I was also considering on changing the text on the button within the popper to "Revert changes" since that is more widely used in other places too.

Reply:
I was also thinking of changing the text on the button within the prompt popper to "revert changes" since it's more widely used in other places as well 🤔

Reply:
this question also sort of applies to situations like while this message (revert all changes) shows the count (2 changes like the example ☝️ ) there are situations (like in arrays when you want to revert the whole array - image) where we don't show the count but the message is pretty similar. 

<img width="370" alt="image" src="https://github.com/sanity-io/sanity/assets/6951139/cdac996d-2f07-438e-afda-fdb65641ed84">

DX wise I can see the argument from both sides where it's annoying having to create so many translations for such similar prompts (and expensive depending on how many languages companies pay translation agencies for) but also it might make sense to keep them as separate things?

Reply:
@mikolajdobrucki just to double check: the path we're choosing is then to change to "revert all" such as the image, yes?

<img width="667" alt="image" src="https://github.com/sanity-io/sanity/assets/6951139/7f914c32-9f3d-48cd-ac87-206aa5364a1f">